WoWsShipBuilder.Common/Infrastructure/DataTransfer/AppNavigator.cs

```diff
@@ -0,0 +1,125 @@
+﻿using System.ComponentModel;
+using Microsoft.AspNetCore.Components;
+using MudBlazor;
+using WoWsShipBuilder.Infrastructure.Metrics;
+
+namespace WoWsShipBuilder.Infrastructure.DataTransfer;
```

Comment:
I would still consider this as a feature, not as infrastructure. Consider moving the entire DataTransfer namespace to the features directory